dashboard/test/controllers/api_controller_test.rb

```diff
@@ -30,6 +30,9 @@ class ApiControllerTest < ActionController::TestCase
     @student_flappy_1 = create(:follower, section: @flappy_section).student_user
     @student_flappy_1.backfill_user_scripts
     @student_flappy_1.reload
+
+    # Seed the RNG deterministically.
+    srand 1
```

Comment:
For the anonymous assessment question. Note that I moved it from that specific test to the setup. Happy to put it back.